src/components/common/Header/Header.tsx

```diff
@@ -1,5 +1,282 @@
+import { theme } from '@styles/theme';
+import styled from 'styled-components';
+import Logo from '../../../assets/mainLogo.svg';
+import CartIcon from '../../../assets/shoppingBag.png';
+import HomeIcon from '../../../assets/home.png';
```

Comment:
제가 craco로 alias 설정해놔서 상대경로 말고 @ 붙여서 경로 설정해주세요~!
```suggestion
import Logo from '@assets/mainLogo.svg';
import CartIcon from '@assets/shoppingBag.png';
import HomeIcon from '@assets/home.png';
```